test-app/tests/integration/components/welcome-page-test.ts

```diff
@@ -20,13 +20,24 @@ module('Integration | Component | welcome-page', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders', async function (assert) {
-    await render(hbs`<WelcomePage/>`);
-
-    await a11yAudit();
+    await render(hbs`
+      <WelcomePage />
+    `);
 
-    assert.ok(true, 'We passed the accessibility audit.');
+    assert
+      .dom('.tomster img')
+      .hasAttribute(
+        'alt',
+        'Under construction',
+        'The image has an alternate text.'
+      )
+      .hasAttribute(
+        'src',
+        '/ember-welcome-page/images/construction.png',
+        'The image source is correct.'
+      );
 
-    const links = findAll('a');
+    const links = findAll('ul a');
```

Comment:
`<EmberWelcomePage>` renders a few other links besides the links to the Quick Start and the Tutorial. To clarify which links are of interest in the test, I wrote `ul a` instead.